src/rculfhash.c

```diff
@@ -568,6 +569,50 @@ unsigned int cds_lfht_fls_ulong(unsigned long x)
 #endif
 }
 
+static void *cds_lfht_malloc(void *state __attribute__((unused)),
+		size_t size)
+{
+	return malloc(size);
+}
+
+static void *cds_lfht_calloc(void *state __attribute__((unused)),
+		size_t nmemb, size_t size)
+{
+	return calloc(nmemb, size);
+}
+
+static void *cds_lfht_realloc(void *state __attribute__((unused)),
+		void *ptr, size_t size)
+{
+	return realloc(ptr, size);
+}
+
+static void *cds_lfht_aligned_alloc(void *state __attribute__((unused)),
+		size_t alignment, size_t size)
+{
+	void *ptr = NULL;
+	if (posix_memalign(&ptr, alignment, size))
```

Comment:
Remove "= NULL".
Add newline between variable and code.
Remove newline before "return ptr;"

Reply:
Fixed.